src/yoroi-wallets/types/types.ts

```diff
@@ -63,13 +63,13 @@ export type YoroiNft = {
   id: string
   name: string
   description: string
-  image: string
-  thumbnail: string
+  image?: string
+  thumbnail?: string
```

Comment:
```suggestion
  image: string | undefined
  thumbnail: string | undefined
```

Reply:
I'd recommend rename `image` to `logo` and transform `thumbnail` in a helper function I.e `getThumbnail(image)` -> `uri&img-width=40&img-height=40` smth like, it will make simpler to merge NFT <> TokenInfo